gradescopeCronJob/gradescope_to_spreadsheet.py

```diff
@@ -73,6 +73,8 @@
 # Tracking the number of_attempts to_update a sheet.
 number_of_retries_needed_to_update_sheet = 0
 
+requests = []
```

Comment:
Can you add a docstring/comment on the overview of the whole process either here or in one of the functions?
Example (I am not sure if this is fully correct):
If len(sys.arg) == 0
1. Query students grades from gradescope for all assignments
2. Store google sheets write requests in this list
3. Write all grades to Google Sheets in one batch update


else if len(sys.arg) > 1:
 [...]

Reply:
This is mostly correct! The script also creates new subsheets for assignments not already present in the sheet. You are right that a docstring would be a useful addition. I will add documentation when I am able (I won't be able to today). I can do that after it is merged in.